g2p_registry_rest_api/services/group_helper.py

```diff
@@ -0,0 +1,56 @@
+import json
+
+from odoo import models
+
+
+class GroupApiHelper(models.AbstractModel):
```

Comment:
Why is this a model? It does not seems to be used as a model.

Reply:
True. `GroupApiHelper` and `IndividualApiHelper` could be a AbstractConnectors, (maybe can be reordered into a `utils` directory, or `helper`). Because its just helping the Group Api service. But there was some issue, making this AbstractConnector and reusing that in the Group Api Service. Hence made this as AbstractModel temporarily.

Reply:
See my other comment by mixin might be better.